datahost-ld-openapi/hurl-scripts/int-012.hurl

```diff
@@ -55,13 +55,12 @@ Accept: text/csv
 
 HTTP 200
 
+## TODO: revision CSV metadata
```

Comment:
Looks like we remove ability to fetch metadata for revisions. That probably should be turned into  a followup issue rather than just a TODO.

Reply:
I will create the followup issue. Good point 👍🏼 

Regarding the "removal" of the ability to fetch Revision metadata: it was never there. The Hurl test was bogus - it only received ever fake data.